realm/src/main/java/io/realm/Realm.java

```diff
@@ -1408,9 +1410,14 @@ public void removeAllChangeListeners() {
     }
 
     void sendNotifications() {
-        List<RealmChangeListener> defensiveCopy = new ArrayList<RealmChangeListener>(changeListeners);
-        for (RealmChangeListener listener : defensiveCopy) {
```

Comment:
I guess removing the defensive copy is OK since Realm instances are thread local anyway?